### ISSUE TITLE
nuxt-security module prevents '@nuxtjs/tailwindcss' module from applying styles

### ISSUE THREAD

**Issue by dotpxg** [Baroshem/nuxt-security#152]
I am having an issue with the nuxt-security module. I am using Nuxt 3. When I add the 'nuxt-security' module to my Nuxt application, the '@nuxtjs/tailwindcss' module's styles are not being applied and gets disabled. I have tried changing the order of the modules in the modules array in the nuxt.config.ts, but the problem persists. When I remove 'nuxt-security' from nuxt.config, tailwind css start working again. I have also checked the code in my nuxt.config.ts file for any mistakes, but I cannot find any. I believe that this is a bug in the nuxt-security module.

Here are the steps to reproduce the issue:

Install Nuxt3 with '@nuxtjs/tailwindcss' module.
Install nuxt-security module
Add the modules to the modules array in your nuxt.config.js file.
Restart your Nuxt application.
Visit your Nuxt application in the browser.
You will see that the '@nuxtjs/tailwindcss' module's styles are not being applied.

**Comment by Baroshem** [comment]
Hey,

thanks for rising this issue.

Taking a look at the description I am not quite sure what can be the issue.

Could you please create a stackblitz reproduction with your exact configuration? I will take a look at it and how I can make it work.

You can use the stackblitz in the readme :)

**Comment by dotpxg** [comment]
Thank you for your attention to this issue.  happy to inform you that I have managed to resolve the problem.

I discovered that the issue was caused by a conflict with the HSTS module I had installed and added to the modules block in my nuxt.config.js file, but without adding the security config.
Initially, the site broke, and tailwindcss failed to load. However, **after adding the SECURITY CONFIGURATION** and running the development server again, I'm pleased to report that the site is now working perfectly. Both tailwindcss and HSTS are functioning as expected.

**_Once again, thank you brother for your prompt reply.. Really appreciate it :)_**
